.gitignore

```diff
@@ -1,4 +1,7 @@
+.idea
```

Comment:
`.idea` is specific to your environment, isn't it? it should be in your global .gitignore, no in the one of the project :) https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
I don't mind treating this as a global to my environment, though I doubt others will do the same.
It's created by PHPStorm.

Reply:
Others should do the same. Isn't it one of the Git best practices? The project `.gitignore` should only ignore files and folders specific to the project. `.idea`, `.DS_Store`, `*.sublime-workspace`, all of those are specific to the user's environment and should be ignored in the global, user-specific `.gitignore`, shouldn't they?